src/main/java/org/springframework/samples/petclinic/rest/JacksonCustomVisitSerializer.java

```diff
@@ -58,6 +58,10 @@ public void serialize(Visit visit, JsonGenerator jgen, SerializerProvider provid
 		}
 		jgen.writeStringField("date", formatter.format(visit.getDate()));
 		jgen.writeStringField("description", visit.getDescription());
+		if (visit.getAdHoc() != null) jgen.writeBooleanField("adHoc", visit.getAdHoc());
```

Comment:
This one line formatting is not complient with the method general format and style